src/ingestion_data.py

```diff
@@ -16,6 +25,16 @@ def __init__(self, json_data):
         self._payload = json_data
         self._sec = None
         self._update_sec = None
+        self._cves = self._get_cves()
+        if self._payload['body']:
+            self._payload['body'] = self._payload['body'][0:MAX_STRING_LENGTH]
+
+    def _get_cves(self):
+        """Get the CVEs mentioned from title or body."""
+        txt = self._payload['title']
```

Comment:
```suggestion
        title = self._payload['title']
```

Reply:
As same variable we are appending body as well, making `title` don't seems to right.